pkg/kubelet/nodestatus/setters_test.go

```diff
@@ -1290,11 +1295,70 @@ func TestVersionInfo(t *testing.T) {
 					},
 				},
 			},
+			kubeProxyVersion: false,
+		},
+		{
+			desc: "DisableNodeKubeProxyVersion FeatureGate enable, versions set in node info",
+			node: &v1.Node{},
+			versionInfo: &cadvisorapiv1.VersionInfo{
+				KernelVersion:      "KernelVersion",
+				ContainerOsVersion: "ContainerOSVersion",
+			},
+			runtimeType: "RuntimeType",
+			runtimeVersion: &kubecontainertest.FakeVersion{
+				Version: "RuntimeVersion",
+			},
+			expectNode: &v1.Node{
+				Status: v1.NodeStatus{
+					NodeInfo: v1.NodeSystemInfo{
+						KernelVersion:           "KernelVersion",
+						OSImage:                 "ContainerOSVersion",
+						ContainerRuntimeVersion: "RuntimeType://RuntimeVersion",
+						KubeletVersion:          version.Get().String(),
+					},
+				},
+			},
+			kubeProxyVersion: true,
```

Comment:
Sorry, when I suggested renaming this field, I meant you should also flip it around so `kubeProxyVersion: false` means "don't set KubeProxyVersion" (ie, enable the feature gate) and `kubeProxyVersion: true` means "_do_ set KubeProxyVersion" (ie, leave the feature gate disabled). So each of the `kubeProxyVersion` values should be flipped, and the `SetFeatureGateDuringTest` call below should set the gate to `!tc.kubeProxyVersion`.